### PR TITLE
docs: update local dev steps, unify permissions in config

### PR DESCRIPTION
**Pull request by bobheadxi**
# Pull Request

## Description

**Give a brief description of your changes:**

Working my way through the local dev guide and a few things have changed on the Slack side of things

## Testing

**If testing this change requires extra setup, please document it here:**

## Ticket(s)

n/a

(Create a copy of that line for each Github Issue affected,
and replace "Affects" with "Closes" if merging this will close the relevant ticket.)

### CODE REVIEW

File: docs/Config.md [hunk 1]
```diff
@@ -12,9 +12,19 @@ your slack app (api.slack.com/apps).
...
 
+The following permission scopes are required:
+
+- `channels:read`
+- `channels:manage`
+- `chats:write`
+- `incoming-webhooks`
+- `users.profile:read`
+- `users:read`
+- `commands`
```
Comment (bobheadxi) [inline docs/Config.md:26]: atm these are just best guesses, not sure what is actually needed

Comment (cheukyin699) [reply docs/Config.md:39]: Really good guesses. Also add:

- `groups:read` (for private channels, optionally, for the random channel promoter)
- `im:write` (for chatting with the people who just joined on how to get set up)

File: docs/LocalDevelopmentGuide.md [hunk 2]
```diff
@@ -76,82 +89,79 @@ We send our logs to CloudWatch for easier storage and querying.
...
+All this can be done from a GitHub organization's Settings > GitHub Apps page.
 
-Under "Private keys", click "Generate a new private key". This will generate
-and allow you to download a new secret key for Rocket 2. Save this to the
-`credentials/` directory as `github_signing_key.pem` - it should already be in
-the PEM file format, bracketed by:
+In the GitHub app's settings, go to "Private keys" and click "Generate a new
+private key". This will generate and allow you to download a new secret key for
+Rocket 2. Save this to the `credentials/` directory as `gh_signing_key.pem` -
+it should already be in the PEM file format, bracketed by:
```
Comment (cheukyin699) [inline docs/LocalDevelopmentGuide.md:111]: This seems out of date. The current way to do things would be to shove the GitHub Key as an environmental variable in your `.env`.

File: docs/LocalDevelopmentGuide.md [hunk 3]
```diff
@@ -76,82 +89,79 @@ We send our logs to CloudWatch for easier storage and querying.
...
+[GitHub webhook configuration docs](Config.md#github-webhook-endpt) for the
+required subscriptions.
 
-- Organization members
+## 6: Set Up Configuration
 
-After doing so, please check the checkboxes below:
+Our repo already contains `sample-env`, the main environmental configuration
+file for the entire app, as well as the `credentials/` directory, where you
+will put credential files like the GitHub app private key.
```
Comment (cheukyin699) [inline docs/LocalDevelopmentGuide.md:133]: As above, we don't use the `credentials` directory.

**Review by chuck-sys — CHANGES_REQUESTED** [review]
Looks good in general. But man do the docs need updating. We also might need to update recommendations on dev practices (that may or may not coincide with "best practices) regarding running using docker vs  pipenv, and how docker may or may not interact badly with local ddb.

File: docs/LocalDevelopmentGuide.md (hunk 2 above)
Comment (bobheadxi) [reply docs/LocalDevelopmentGuide.md:111]: This is still right, the first step is to download it

File: docs/LocalDevelopmentGuide.md (hunk 3 above)
Comment (bobheadxi) [reply docs/LocalDevelopmentGuide.md:133]: Will be used in #484 - it's handy as a place to put credentials in the meantime

File: docs/Config.md (hunk 1 above)
Comment (bobheadxi) [reply docs/Config.md:39]: 98da49f

**Review by chuck-sys — APPROVED** [review]
lgtm

**Pull request merged**